projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts

```diff
@@ -1,14 +1,19 @@
+import * as authentication from './auth-forms';
+
 const nextGroupButtonSelector =
   'cx-config-previous-next-buttons div div:last button';
 const previousGroupButtonSelector =
   'cx-config-previous-next-buttons div div:first button';
 const addToCartButtonSelector = 'cx-config-add-to-cart-button button';
 
-export function clickOnConfigureButton() {
+const email = 'cpq02@sap.com';
+const password = 'welcome';
```

Comment:
I don't know if it's ok to put the PW here (even if it is a test user). There was a bigger exercise to remove all passwords from the commerce code base recently. How do other SPA e2e tests handle this?